ooni/deck.py

```diff
@@ -171,11 +166,122 @@ def setup(self):
 
         if self.bouncer:
             log.msg("Looking up collector and test helpers")
-            yield self.lookupCollector()
+            yield self.lookupCollectorAndTestHelpers()
+
+
+    def sortAddressesByPriority(self, priority_address, alternate_addresses):
+        onion_addresses= []
+        cloudfront_addresses= []
+        https_addresses = []
+        plaintext_addresses = []
+
+        if priority_address.startswith('httpo://'):
```

Comment:
Yes, I guess we can add support for both, but we need to support the `httpo` schema since it's used everywhere in ooniprobe to specify tor hidden service addresses.